src/AutorestSwift/View Models/PackageFileViewModel.swift

```diff
@@ -29,13 +29,13 @@ import Foundation
 /// View Model for an the Package.swift file.
 struct PackageFileViewModel {
     let name: String
-    // FIXME: This should come from config?
-    let namespace = "secret"
-    let version = "0.1"
+    let namespace: String
+    let version: String
 
     init(from model: CodeModel) {
-        // FIXME: Blunt-instrument to override the package name from the code model
-        let commandLineArguments = CommandLineArguments()
-        self.name = Manager.packageName ?? commandLineArguments["--package-name"] ?? model.packageName
+        let config = Manager.shared.config
+        self.name = config.packageName ?? model.packageName
+        self.namespace = config.namespace ?? ""
+        self.version = config.packageVersion ?? ""
```

Comment:
Should we failed the code generation if there is no values for 'namespace' and 'version'? instead of setting them to empty string and continue?

Reply:
Possibly. We were hard-coding them to "secret" and "0.1" so I don't think these values are currently used anywhere in our codegen.